lazy_tensor_core/lazy_tensor_core/csrc/tensor_aten_ops.cpp

```diff
@@ -457,7 +457,8 @@ LazyTensor tanh_backward(const LazyTensor& grad_output,
   // Shape stays the same since pow is a unary op
   std::vector<torch::lazy::Shape> shapes{output.shape().Get()};
   torch::lazy::NodePtr pow_node =
-      torch::lazy::MakeNode<ir::ops::PowTensorScalar>(output.GetIrValue(), 2,
+      torch::lazy::MakeNode<ir::ops::PowTensorScalar>(output.GetIrValue(), 
+                                                      LazyGraphExecutor::Get()->GetIrValueForScalar(2, output.GetDevice()),
```

Comment:
Should this one be using the 'FromCodegen' variant?

I believe it should as long as PowTensorScalar does not expand it before storing it in the IR.